animalkingdom/src/animalkingdom/Fish.java

```diff
@@ -0,0 +1,38 @@
+package animalkingdom;
+
+// if you want to see whats happening here go to mammels, its the same thing
+
+public class Fish extends AnimalActions {
+	
+	public Fish(String name, int year, int food) {
+
+		this.name = name;
+		this.year = year;
+		this.food = food;
+	}
+
+	@Override
+	public String getName(){
+		return name;
+	};
+
+	@Override
+	public String getMoveType(){
+		return "Fins";
+	};
+
+	@Override
+	public String getBreatingType(){
+		return "gills";
+	};
+
+	@Override
+	public String getReproductionType(){
+		return "lays small eggs";
+	};
```

Comment:
It seems that you forgot to define a `public abstract String getReproductionType()`  method in your abstract `AnimalActions` class - without doing so, the compiler will run into an error upon compiling due to it getting confused about what method it is supposed to be overriding.